joint_space_controller:
  ros__parameters:
    type: tsid_controllers/JointSpaceTsidController
    update_rate: 25
    
    joint_state_names:
      - torso_lift_joint
ยก

    joint_command_names:
      - torso_lift_joint



    interface_name: position

    velocity_scaling: 1.0
    posture_gain: 1.0
    